apps/sample-imdb/pages/api/search.ts

```diff
@@ -0,0 +1,19 @@
+import { type NextApiRequest, type NextApiResponse } from 'next'
+import { searchMovies } from '~/lib/db.server'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { term } = req.query
+
+  if (Array.isArray(term) || !term) {
+    res.status(402).json({ message: 'term must be of type string' })
```

Comment:
```suggestion
    res.status(422).json({ message: 'term must be of type string' })
```

<img width="511" alt="image" src="https://user-images.githubusercontent.com/1761469/199226378-c54be3a3-e066-4a24-929e-271a13a8084b.png">

I will be surprise if we ask for money here 😁